main.cpp

```diff
@@ -580,13 +655,13 @@ GameStore::~GameStore()
     std::cout << "Game store demolished" << std::endl;
 }
 
-void sellControllersWithXbox(Xbox, int numControllersToSell)
+void sellControllersWithXbox(Xbox&, int numControllersToSell)
```

Comment:
pass by `const reference` if you're not modifying the passed-in object
- [ ] revise